doc/contribute/build_setup.rst

```diff
@@ -53,3 +53,56 @@ If you use `Visual Studio Code`_ you can install the `Remote Container`_ extensi
 .. _`Visual Studio Code`: https://code.visualstudio.com/
 .. _`Remote Container`: https://code.visualstudio.com/docs/remote/containers
 
+Develor for Azure Linux OS
+--------------------------
+
+Setting up Azure Linux VM
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See the official Azure Linux documentation for developers for details: `Azure Linux VMs in Azure | Mariner OS <https://eng.ms/docs/products/azure-linux/gettingstarted/azurevm/azurevm>`_.
```

Comment:
That does not seem like a public link, so I don't think we should include it here.